README.md

```diff
@@ -1,265 +1,85 @@
 
-## Deploying CDI
+## Deploy it
 
-### Assumptions
-- A running Kubernetes cluster with roles and role bindings implementing security necesary for the CDI controller to watch PVCs and pods across all namespaces.
-- A storage class and provisioner.
-- An HTTP or S3 file server hosting VM images
+Deploying the CDI controller is straightforward. In this document the _default_ namespace is used, but in a production setup a [protected namespace](#protecting-the-golden-image-namespace) that is inaccessible to regular users should be used instead.
 
-- An optional "golden" namespace acting as the image registry. The `default` namespace is fine for tire kicking.
+  ```
+  $ VERSION=<cdi version>
+  $ kubectl create -f https://github.com/kubevirt/containerized-data-importer/releases/download/$VERSION/cdi-operator.yaml
+  $ kubectl create -f https://github.com/kubevirt/containerized-data-importer/releases/download/$VERSION/cdi-operator-cr.yaml
+  ```
 
-### Download CDI
+If you're running OpenShift, make sure you enable privileged containers
 
-`$ go get kubevirt.io/containerized-data-importer`
+  ```
+  $ oc adm policy add-scc-to-user privileged -z cdi-sa
+  ```
 
-*Or*
-
-`$ git clone https://github.com/kubevirt/containerized-data-importer.git $GOPATH/src/kubevirt.io/containerized-data-importer`
-
-*Or download only the yamls:*
-
-```shell
-$ mkdir cdi-manifests && cd cdi-manifests
-$ wget https://raw.githubusercontent.com/kubevirt/containerized-data-importer/kubevirt-centric-readme/manifests/example/golden-pvc.yaml
-$ wget https://raw.githubusercontent.com/kubevirt/containerized-data-importer/kubevirt-centric-readme/manifests/example/endpoint-secret.yaml
-```
-
-### Deploy CDI from a release
-
-Deploying the CDI controller is straight forward. In this document the _default_ namespace is used, but in a production setup a [protected namespace](#protecting-the-golden-image-namespace) that is inaccessible to regular users should be used instead.
-
-1. Ensure that the cdi-sa service account has proper authority to run privileged containers, typically in a kube environment this is true by default.
-   If you are running an openshift variation of kubernetes you may need to enable privileged containers in the security context:
-
-```
-$ oc adm policy add-scc-to-user privileged -z cdi-sa
-```
+## Use it
 
-2. Deploy the controller from the release manifest:
+Create a PVC and load data from an http source on to it (processes a tgz file for you)
 
 ```
-$ VERSION=<cdi version>
-$ kubectl create -f https://github.com/kubevirt/containerized-data-importer/releases/download/$VERSION/cdi-controller.yaml
+$ kubectl create -f https://raw.githubusercontent.com/kubevirt/containerized-data-importer/release-v1.5/manifests/example/import-archive-pvc.yaml
```

Comment:
I just released 1.6, so this is already out of date, I am not particularly fond of the way it was before, but that didn't lock us into a particular version to keep up with.

Reply:
I'll use the same idea of just going to the local directory instead to avoid that; the other option I considered was $VERSION; but maybe it's reasonable to consider this README being geared towards working from source locally?

My only concern is that we are introducing dependencies in the README for changes to those files in the future, which setting a version solves; but meh.

Reply:
Well if I understand the intent of that particular example is to show this working for a released version of CDI, and thus should not be pointing to master, but the latest release. I am not sure if there is a good way to not paint ourselves into a particular version corner. I know github has a rest endpoint that allows you to discover the latest release, so with some create curl command you should be able to get the latest version from that endpoint, and then use that to grab the example. Not sure.